File: C/1020.c
#include <stdio.h>
 
int main() {
 
    int aux;
    scanf("%d", &aux);
    printf("%d ano(s)\n", (aux/365));
    aux %= 365;
    printf("%d mes(es)\n", (aux/30));
    printf("%d dia(s)\n", (aux%30));
 
    return 0;
}

File: Javascript/1020.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');

let aux = parseInt(lines.shift());
console.log("%d ano(s)", parseInt(aux/365));
aux %= 365;
console.log("%d mes(es)", parseInt(aux/30));
console.log("%d dia(s)", parseInt(aux%30));